src/utils/ExtractTestResults.ts

```diff
@@ -34,23 +38,25 @@ export const extractMCTestResults = (record: DynamoDBRecord): MCRequest[] => {
     )
     .filter(
       (testType) =>
-        testType.testResult === 'pass' || testType.testResult === 'prs',
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-enum-comparison
```

Comment:
I'd prefer if you came up with a solution without switching off the listing rules

Reply:
I would typically agree but for some reason the eslint was throwing errors at implementations we've used in other repos, and this one seems to have more ignores in it than elsewhere i've seen maybe for that reason. Not sure though, thoughts?